### PR TITLE
Fix ssl issue

### PR DESCRIPTION
**Pull request by devmarcy**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CMC-1275

### Change description ###

- Added in a /certificate-bundle directory
- Changed ```run.sh``` and added a line that will copy the certificate in the /certificate-bundle directory and put it in the pact library, overwriting the certificate file there.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/app.ts [hunk 1]
```diff
@@ -16,12 +15,11 @@ const { setupDev } = require('./development');
 
 const env = process.env.NODE_ENV || 'development';
 const developmentMode = env === 'development';
-
 export const app = express();
 app.locals.ENV = env;
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
```
Comment (sabahirfan) [inline src/main/app.ts:20]: do we still need this?

File: package.json [hunk 2]
```diff
@@ -57,7 +57,8 @@
     "tsconfig-paths": "^3.9.0",
     "typescript": "^4.0.5",
     "@pact-foundation/pact": "^9.11.1",
-    "@sendgrid/mail": "^7.4.0"
+    "@sendgrid/mail": "^7.4.0",
+    "express-http-proxy": "^1.6.2"
```
Comment (sabahirfan) [inline package.json:61]: we can remove this proxy library as we are not using this now.

**Review by sabahirfan — COMMENTED** [review]
@devmarcy will we need to re bundle the certificates, once they are expired?

File: src/main/app.ts (hunk 1 above)
Comment (devmarcy) [reply src/main/app.ts:20]: without this the email send will fail

**Review by sabahirfan — APPROVED** [review]

**Review by douglasrice — APPROVED** [review]

**Pull request merged**